orcid-model/src/main/resources/record_2.0_rc1/samples/education-2.0_rc1.xml

```diff
@@ -3,6 +3,16 @@
 	xmlns:common="http://www.orcid.org/ns/common" xmlns:education="http://www.orcid.org/ns/education"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://www.orcid.org/ns/education ../education-2.0_rc1.xsd ">
+	<common:created-date>2001-12-31T12:00:00</common:created-date>
+	<common:last-modified-date>2001-12-31T12:00:00</common:last-modified-date>
+    <common:source>
+        <common:source-orcid>
+            <common:uri>http://orcid.org/8888-8888-8888-8880</common:uri>
+            <common:path>8888-8888-8888-8880</common:path>
+            <common:host>orcid.org</common:host>
```

Comment:
@lizkrznarich @wjrsimpson @amontenegro The place where created-date, last-modified-date and source has changed to be in a consistent place/order for all activities. My preference would to have those show up last but do to limitations in xsd extends they will be first.